src/main/java/org/shanerx/tradeshop/Updater.java

```diff
@@ -62,11 +64,19 @@
 	
 	private PluginDescriptionFile pdf;
 	private BuildType build;
-  
-        private final URL url = "https://api.spigotmc.org/legacy/update.php?resource=32762"; // Edit API URL.
+
+	private URL url = null;
 
 	public Updater(PluginDescriptionFile pdf) {
 		this.pdf = pdf;
+
+		log = Bukkit.getLogger();
+
+		try {
+			url = new URL("https://api.spigotmc.org/legacy/update.php?resource=32762"); // Edit API URL.
+		} catch (MalformedURLException ex) {
+			log.log(Level.WARNING, "Error: Bad URL while checking {0} !", pdf.getName());
```

Comment:
Maybe use String.format(String, String)

Btw, I completely forgot about that try..catch block.

Reply:
Also, I think we could also use an empty catch block as that URL is a constant and is definitely a valid URL.

Reply:
Should always have an error output just incase, it should never but that doesn't mean it won't and that is the proper way for the url to be done.